battle_test.go

```diff
@@ -42,12 +42,15 @@ func TestBattleOneRound(t *testing.T) {
 	party1 := Party{
 		Agent: &a1,
 	}
+	pound := GetMove(1)
 	pkmn1 := GetPokemon(4)
+	pkmn1.Moves[0] = &pound
```

Comment:
Can we make this a `pokemon.SetMove(i int, move *Move)`?

Reply:
Yeah, I agree, but it should go in another PR imo